### DISCUSSION TITLE
AI in 5 or 10 years

### DISCUSSION THREAD

**Discussion by paularamo** [openvinotoolkit/openvino_notebooks#665] [Show and tell]
Let us know how you see AI in 5 to 10 years. And what role you would like to play in that journey.

**Comment by TrentonWallis** [comment]
I think AI and ML will lead the way in embedded system configuration to increase optimization of microcontrollers. 

I hope to work in an industry on the leading edge of embedded systems and put the good work in to advance MLs capabilities in the embedded systems field.

**Comment by anuPh23** [comment]
I want to see myself contributing towards creating more representative-of-different-subpopulations data, which in turn will produce responsible ML and AI based systems.

**Comment by tomas19** [comment]
I hope IA/ML to be incorporated in the prediction of coastal hazards to help communities to better prepared for them, and I will really like to contribute to it.

**Comment by Morsus98** [comment]
I foresee ML devices expand into the embedded space, augmenting traditional sensors and replacing much of the human component in many different fields, such as agriculture.

**Comment by marosasn** [comment]
I see a world that is more efficient due to ML!! I see better technology and applications that will allow us to solve problems that we've not been able to solve due to our human limitations. I hope to be a part of this!

**Comment by esogea** [comment]
I think in 5 to 10 years significant progress may have been made towards a generalized AI model. Tools like DALL-E will be more integrated into white-collar workers' workflows and we will continue to be reliant on AI algorithms in our daily lives.

I would like to play a part in developing the algorithms that shape everyday life.

**Comment by vamshi-chidara** [comment]
I am confident of AI expanding more into embedded technology space. Self driving cars will be more common than ever. We will have better and faster AI and ML technology to help moving stock from warehouse to customers in the supply chain, predict and help in taking care of serious health issues, climatic calamities among others. I hope I will be in a position to contribute to this expansion and development in 5 years.

**Comment by sbgadhwala** [comment]
In the coming 5 to 10 years, technologies like AI Ops would be dominant in the fields of obtaining medical excellence like disease detection, create effective vaccine and its prediction, and integrating with robotics, the tasks involving humans for "dangerous" and "decision making" field work would be next to eliminated. These are just a glimpse of what would be AI is capable of.

**Comment by MEKNC3** [comment]
I think AI will dominate our daily lives in 10 years.  The growing dependency on technology will free up humanity to pursue more interesting tasks

**Comment by anschoon** [comment]
AI and ML are slowly being integrated into every system.  I expect both to be more integrated into projects as we move forward.  
Particularly I see it being used to make more accurate predictions using modeling.  In my field of bioinformatics, I am seeing more and more researchers using visual models to predict results of an experiment long before its conclusion.  In addition, pattern predictions are being used to select important genomic regions for trait identification and protein regulation.  I expect both branches to become more integrated into the field and better models and more accurate predictions to come out in next several years.  Which would help with experiment timing and costs as possible results have already gone through preprocessing.  I hope to be a part of this expansion and development.

**Comment by KStickney** [comment]
I think AI and ML will help increase production and optimization in agriculture. I would like to work in a field using AI drones and robots for small farms.

**Comment by Patjim22** [comment]
I think AI and ML will automate our fields and allow for autonomous package delivery

**Comment by nathankohen** [comment]
I see AI technologies being heavily involved in automating product development. Through large amounts of data previously collected, teams will be able to develop new product lines that optimize performance for desired properties. I see this spreading to nearly all industries, and I would love to contribute to this initiative!

**Comment by vikrampande7** [comment]
AI will become handier for manufacturers who are looking to automate specific complex tasks with assistance from robots to be able to optimize their operations.

**Comment by Sumukha21** [comment]
I think AI will be able to automate manufacturing in the industry. I think we will have robust level 4 autonomous driving realized. I hope AI in the medical domain can be developed to support doctors in providing treatment in a more efficient and effective manner. I want to contribute towards realizing fully autonomous driving.

**Comment by derekchang4** [comment]
I think AI will play a huge role in more intelligent technologies in the future, further expanding their capabilities. I see it playing a bigger part in everyone's daily lives as things like Dall-E become more mainstream. I hope to learn how it works, learn how to use it, and maybe work with it in the future!

**Comment by SaiPrasanthBL** [comment]
I think AI might open many economic opportunities to millions around the globe, it might even help bring people out of poverty, AI might cause a debate on employment but it will also act as a leveler creating new jobs such as AI developers.

**Comment by Mdmoffit** [comment]
I think AI will contribute to increasingly more efficient and intelligent processing power and will only exponentially improve with time. I hope to work with AI in the future in the automotive field specifically autonomous driving and improving safety.

**Comment by mjmathew7** [comment]
I believe that AI will play a large role in defining the future. I see this happening in automated product development, the improvement in information technologies, and more.

**Comment by brmarkus** [comment]
I fear ethical aspects of AI will be less and less recognized.

**Comment by AnishaUdayakumar** [comment]
AI is constantly evolving and changing our world. It certainly has a lasting impact on just about every industry imaginable. We’re already seeing it in our smart devices, cars, healthcare system and daily apps, and we’ll continue to see its influence permeate deeper into many other industries for the foreseeable future with AI ethics in place.
